src/rpc/misc.cpp

```diff
@@ -44,7 +44,7 @@ UniValue getinfo(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 0)
         throw runtime_error(
             "getinfo\n"
-            "Returns an object containing various state info.\n"
+            "\nDEPRECATED. Returns an object containing various state info.\n"
```

Comment:
Well, people use the help system. It's the only up-to-date RPC documentation that there is, and errors are usually found quickly.

But maybe not for old calls. Of course it may make sense to mention it in the release notes as well. I don't think we ever did, but even if so a reminder probably can't hurt.